packages/esm-patient-registration-app/src/patient-registration/field/field.scss

```diff
@@ -18,9 +18,9 @@
 }
 
 .grid {
-  display: grid;
-  grid-template-columns: 1fr 1fr;
-  column-gap: spacing.$spacing-05;
+  display: flex;
+  flex-direction: column;
```

Comment:
If it's a column and no `justify-content` or align-items, I believe we can totally remove it.